persistence/src/commonMain/db_user/com/wire/kalium/persistence/Converations.sq

```diff
@@ -8,7 +9,9 @@ CREATE TABLE Conversation (
     team_id TEXT,
     mls_group_id TEXT,
     mls_group_state TEXT AS ConversationEntity.GroupState NOT NULL,
-    protocol TEXT AS ConversationEntity.Protocol NOT NULL
+    protocol TEXT AS ConversationEntity.Protocol NOT NULL,
+    has_unnotified_messages INTEGER AS Boolean DEFAULT 0,
```

Comment:
I would consider replacing `has_unnotified_messages` with a `last_modified_date`. This you would also update this date whenever we persist a new message but it's more generic (can be used to sort the conversation list). You could then retrieve the conversations with un-notified message by `last_modified_date  > last_notification_date`. Another advantage of this approach is that you don't need "reset" the `last_modified_date` like you do for the `has_unnotified_messages`.

Reply:
that is a really cool idea!